<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Props Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="ui_list_widget"/>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="ui_file_edit"/>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="ui_save_btn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="ui_refresh_btn">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="ui_open_btn">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="ui_insert_btn">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ui_name_lbl">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="ui_close_btn">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="ui_sort_cbox">
         <item>
          <property name="text">
           <string>Date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2" colspan="3">
        <widget class="QListWidget" name="ui_info_widget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
